en/6-use-open-standards-solutions.md

```diff
@@ -73,8 +115,17 @@ Using open standards and common government platforms helps you to:
 
 **[TODO: Add/revise checklist items]**
 
+- For all software acquisition, open source software must be actively and fairly considered alongside proprietary ones **(SSC Draft Position on OSS (GC))**
+- Where there is no significant overall (full-lifecycle) cost difference between open source and closed-source products, open source will be selected on the basis of its additional inherent flexibility **(SSC Draft Position on OSS (GC))**
+- OSS support models and options must be evaluated and selected **(SSC Draft Position on OSS (GC))**
+- All software, including OSS, must have a defined and managed lifecycle that is supported by a robust toolset (software acquisition and distribution guidelines) **(SSC Draft Position on OSS (GC))**
+

```

Comment:
This blank line will result in a second list. Is that intentional? Are you trying to group the items separately?